libs/utils/env.py

```diff
@@ -503,6 +497,25 @@ def _init_target(self, force = False):
                 raise RuntimeError('Failed to initialized [{}] module, '
                         'update your kernel or test configurations'.format(module))
 
+    def install_tools(self, tools):
+        """
+        Install tools additional to those specified in the test config
+
+        :param tools: The list of names of tools to install
+        :type tools: list(str)
+        """
+        tools_to_install = []
+        for tool in tools:
+            if tool in self.__installed_tools:
+                continue
+            binary = '{}/tools/scripts/{}'.format(basepath, tool)
+            if not os.path.isfile(binary):
+                binary = '{}/tools/{}/{}'\
+                         .format(basepath, self.target.abi, tool)
+                tools_to_install.append(binary)
+        self.target.setup(tools_to_install)
```

Comment:
But still we expose Env to a strict dependency on a devlib method implementation.
Not sure it's safe to assume that that function will never integrate code meant to be executed just one time...